ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2

```diff
@@ -193,6 +194,12 @@ host_monitoring_cron:
 - name: run docker dns test
   minute: "*/10"
   job: source $HOME/.bashrc; ops-runner -f -s 60 -n csddr.docker.dns.resolution /usr/bin/cron-send-docker-dns-resolution
+
+
+- name: check status for unresovled banner
+  minute: "*/10"
+  job: pageid={{ g_status_banner_pageid }};authid={{ authid=g_status_banner_authid  }} ops-runner -f -s 60 -n cssbc.status.unresolved.banner  /usr/bin/cron-send-status-banner-check
```

Comment:
```
 +  job: 
pageid={{ g_status_banner_pageid }};
authid={{ authid=g_status_banner_authid  }} 
ops-runner -f -s 60 -n cssbc.status.unresolved.banner  /usr/bin/cron-send-status-banner-check 
```

Is `authid=g_status_banner_authid` the correct instantiation?
Also is: `pageid={}; authid={}` using the semicolon the right invocation? I haven't tested this part myself yet.